src/components/[guild]/claim-poap/components/ClaimPoap.tsx

```diff
@@ -48,9 +48,8 @@ const ClaimPoap = ({ rolePlatformId }: Props) => {
       >
         <ClaimPoapButton
           rolePlatformId={rolePlatformId}
+          rolePlatform={rolePlatform}
```

Comment:
I think we should only provider `rolePlatform` here, because then we can use `rolePlatform.id` inside this component